apps/microtvm/zephyr/template_project/app-overlay/nucleo_l4r5zi.overlay

```diff
@@ -0,0 +1,23 @@
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+&rcc {
+	clock-frequency = <DT_FREQ_M(120)>;
+};
```

Comment:
That is not correct.
You should update the PLL configuration accordingly:
```
&pll {
	div-m = <2>;
	mul-n = <30>;
	div-p = <7>;
	div-q = <2>;
	div-r = <2>;
	clocks = <&clk_hsi>;
	status = "okay";
};
```
Note that the impacts of effects on bus configurations have to be tested.

Reply:
@erwango Thanks a lot. Do you mind to tell how exactly one finds the params (M, N, P, Q, and R) for the PLL? I see them in the clock tree in  RM0432 rev. 9 (Arm Reference Manual), Figure 16, but could not find any formula about how to relate them to the 120 MHz freq.

Reply:
> @erwango Thanks a lot. Do you mind to tell how exactly one finds the params (M, N, P, Q, and R) for the PLL? I see them in the clock tree in RM0432 rev. 9 (Arm Reference Manual), Figure 16, but could not find any formula about how to relate them to the 120 MHz freq.

Core freq = Input freq (16MHz) / M(2) * N(30) / R(2)

Reply:
@erwango Thanks for the explanation. I found also the formula in the Reference Manual, below the bit descriptions for the `RCC_PLLCFGR`, where also there are formulas using the Q and P params. Could you please review https://github.com/apache/tvm/pull/12756 ?